Python/Product/Cookiecutter/View/CookiecutterOptionsPage.xaml.cs

```diff
@@ -76,7 +77,28 @@ class TemplateContextItemTemplateSelector : DataTemplateSelector {
             var element = container as FrameworkElement;
             var p = item as ContextItemViewModel;
             if (element != null && p != null) {
-                var templateName = p.Items.Count == 0 ? "textTemplate" : "listTemplate";
+                string templateName;
+                switch (p.ValueType) {
+                    case Model.ContextItemValueType.String:
+                        templateName = "textTemplate";
```

Comment:
Yeah, after I checked it in I thought I could use the enum value name directly to match the template, but passing through the original string without converting to enum is even less work ;)